benchmark.py

```diff
@@ -20,6 +25,7 @@
 parser.add_argument('-d', '--double', action='store_true')
 options = parser.parse_args()
 
+LIB_EXT = torch.utils.cpp_extension.LIB_EXT
```

Comment:
Looks like all the changes you made in benchmark.py are not used.

Reply:
The op is called from LLTM module class' `forward()` See {cpp|cuda}/lltm.py